src/Dependencies/Composition/project.json

```diff
@@ -0,0 +1,9 @@
+{
+  "supports": {},
+  "dependencies": {
+    "Microsoft.Composition": "1.0.27"
```

Comment:
What is the advantage of this approach for single dependency projects like this?  I get it for the VS binaries, but not Moq and Composition.